relayer/crates/starknet-cli/src/commands/demo/transfer.rs

```diff
@@ -0,0 +1,154 @@
+use std::str::FromStr;
+
+use cgp::prelude::*;
+use hermes_cli_components::traits::build::CanLoadBuilder;
+use hermes_cli_components::traits::command::CommandRunner;
+use hermes_cli_components::traits::output::HasOutputType;
+use hermes_encoding_components::traits::encode::CanEncode;
+use hermes_logging_components::traits::has_logger::HasLogger;
+use hermes_logging_components::traits::logger::CanLog;
+use hermes_logging_components::types::level::LevelInfo;
+use hermes_relayer_components::chain::traits::queries::chain_status::CanQueryChainStatus;
+use hermes_starknet_chain_components::types::channel_id::ChannelId;
+use hermes_starknet_chain_components::types::cosmos::height::Height;
+use hermes_starknet_chain_components::types::cosmos::timestamp::Timestamp;
+use hermes_starknet_chain_components::types::messages::ibc::channel::PortId;
+use hermes_starknet_chain_components::types::messages::ibc::denom::{
+    Denom, PrefixedDenom, TracePrefix,
+};
+use hermes_starknet_chain_components::types::messages::ibc::ibc_transfer::{
+    MsgTransfer, Participant, TransferPacketData,
+};
+use hermes_starknet_chain_context::contexts::encoding::cairo::StarknetCairoEncoding;
+use ibc::core::host::types::identifiers::PortId as IbcPortId;
+use starknet::core::types::Felt;
+
+use crate::contexts::app::StarknetApp;
+
+#[derive(Debug, clap::Parser, HasField)]
+pub struct TransferArgs {
+    /// Amount to transfer
+    #[clap(
+        long = "amount",
+        required = true,
+        value_name = "AMOUNT",
+        help_heading = "REQUIRED"
+    )]
+    pub amount: String,
+
+    /// Denom of the amount to transfer. If not specified this
+    /// will default to `stake`
```

Comment:
Why default to `stake`? In starknet the tokens are erc20 contract address. This has to be a `Felt`.

PS. if you want to keep it optional, you can use the canonical `strk` token address -- this is known and fixed after a network is launched. So you may keep a `const` that is passed here.

You can use [`std::env`](https://doc.rust-lang.org/std/macro.env.html) macro to pass the value at compile time.